src/sherpa_ai/connectors/vectorStorage.py

```diff
@@ -0,0 +1,121 @@
+from langchain.embeddings.sentence_transformer import SentenceTransformerEmbeddings
```

Comment:
This filename is confusingly similar to the existing connectors/vectorstores.py. Please rename one or the other to distinguish them from each other.

Also I notice connectors/vectorStorage.py here, and memory/vector_storage.py.  Use a consistent naming convention.